docs/docs-ref-autogen/office/office.document.yml

```diff
@@ -3552,10 +3557,11 @@ items:
 
 
       *Supported hosts, by platform* <table> <tr><th> </th><th> Office for Windows desktop </th><th> Office Online (in
-      browser) </th><th> Office for iPad </th></tr> <tr><td><strong> Access </strong></td><td> </td><td> Y </td><td>
-      </td></tr> <tr><td><strong> Excel </strong></td><td> Y </td><td> Y </td><td> Y </td></tr> <tr><td><strong>
-      PowerPoint </strong></td><td> Y </td><td> Y </td><td> Y </td></tr> <tr><td><strong> Word </strong></td><td> Y
-      </td><td> Y </td><td> Y </td></tr> </table>
+      browser) </th><th> Office for iPad </th><th> Office for Mac </th></tr> <tr><td><strong> Access </strong></td><td>

```

Comment:
Should the "Supported hosts, by platform" table within the **url** property section also contain a column for "Office for Mac" ?

Reply:
I only have info for methods from the SOT and not properties so I couldn't verify that. I was able to make a best guess for some of the properties (e.g. the platform for that host supports the entire object or the requirement set and so by extension supports any and all methods and properties) but I couldn't make that determination for this property and others like it.

In DefinitelyTyped, I can certainly add the column but I don't have the details needed to fill it for each host.

Reply:
Thanks for the explanation. If we don't have the information necessary to confidently populate the additional column, probably best to omit it for now.